faker/providers/address/el_GR/__init__.py

```diff
@@ -108,14 +108,17 @@ def city(self):
     def region(self):
         return self.random_element(self.regions)
 
+    # FIXME: latlng for el_GR should be moved to geo provider in default en_US locale
     def latlng(self):
```

Comment:
If this is to remain, it should be moved to the top-level geo provider, though its usefulness is dubious since we already have a top-level `geo_coord` for single coordinates, and now `coords_on_land` provides a `coords_only` option which is more useful than this.

Reply:
i think this should remain for backward-compatibility, but it should call the `geo_coord` method.

I'd like for it to also raise a `PendingDeprecation` warning. Ideally, I'd like to manage our deprecations by suing the [`deprecation`](https://deprecation.readthedocs.io/en/latest/) library. But this could be done in a separate PR

Reply:
Your second note: "i think this should remain for backward-compatibility, but it should call the geo_coord method" contradicts your first: "Maybe those methods should be called local_latitude and local_longitude or something like that." I've already made the first change, moving them into geo and renaming with `local_*`. If you feel strongly, I'll undo it and follow your second comment instead. Yes, that means my change is potentially a breaking change for a very small percentage of users, but it's a clean break with less follow-up maintenance hassle in the future and no need to go through a deprecation cycle.